External/FEXCore/Source/Interface/Core/OpcodeDispatcher/Flags.cpp

```diff
@@ -39,6 +39,11 @@ constexpr std::array<uint32_t, 17> FlagOffsets = {
 };
 
 void OpDispatchBuilder::SetPackedRFLAG(bool Lower8, OrderedNode *Src) {
+  // Canonicalize flags early
+  // Could be invalidate if we had masked invalidation
```

Comment:
Is `invalidate` referring to another function name here? If so, quotes or trailing `()` help explain that. Currently, the comment reads like a typo'ed sentence and it's unclear what's tried to be said.

Reply:
Resolved. Updated comment to reflect real function name.